### ISSUE TITLE
[Run Apertium] See why new Apertium tools give a different chunked result

### ISSUE THREAD

**Issue by rmlockwood** [rmlockwood/FLExTrans#612]
In the k-z project in the 'ger and -iyam' text for the line GEN 1: 2, I get different results when transferring the word rûyê. 
With the old Apertium rules I get:
ch_n NP OBLIQUENESS [rî1.1 n ez 2 m]

With the new Apertium tools I get:
ch_n NP OBLIQUENESS [/î1.1 n ez 2 m]

**Comment by mr-martian** [comment]
This looks like https://github.com/apertium/apertium/issues/199

This can probably be fixed by changing
```
${FLEXTOOLS_PATH}\apertium-transfer -t tr.t1x transfer_rules.t1x.bin bilingual.bin ${SOURCE_PATH}
```
to
```
${FLEXTOOLS_PATH}\lt-proc -b bilingual.bin ${SOURCE_PATH} | ${FLEXTOOLS_PATH}\apertium-transfer -t tr.t1x transfer_rules.t1x.bin
```
in the makefile. Invoking `apertium-transfer` with a bilingual dictionary is deprecated.

**Comment by rmlockwood** [comment]
This worked for me except for one of my regression tests where the affix looks like <1/3SG>. Maybe apertium-transfer could handle that in the input file and let it go through, but lt-proc doesn't like it and makes a mess of things?

**Comment by mr-martian** [comment]
blah.dix
```xml
<dictionary>
  <sdefs>
    <sdef n="a/b"/>
    <sdef n="x/z"/>
    <sdef n="n"/>
  </sdefs>
  <section id="main" type="standard">
    <e><p><l>a<s n="n"/><s n="a/b"/></l><r>b<s n="n"/><s n="x/z"/></r></p></e>
    <e><p><l>c<s n="n"/></l><r>d<s n="n"/></r></p></e>
  </section>
</dictionary>
```

```
$ lt-comp lr blah.dix blah.bin
main@standard 5 5
$ echo '^a<n><z>$ ^a<n><a/b>$ ^c<n><a/b>$' | lt-proc -b blah.bin 
^a<n><z>/@a<n><z>$ ^a<n><a/b>/b<n><x/z>$ ^c<n><a/b>/d<n><a/b>$
```
Tags with slashes are behaving as I would expect here. What sort of failure are you getting?

**Comment by rmlockwood** [comment]
Here's some sample data where I get the failure.
Expected result: ^älska1.1<v><INF><IND>$
Actual resuilt: ^3SG><INF><IND>$ 
[issue_612.zip](https://github.com/rmlockwood/FLExTrans/files/15257295/issue_612.zip)

**Comment by mr-martian** [comment]
Ah, yep. When `apertium-transfer` splits the incoming word into source and target, it checks for `/` without seeing whether that `/` is inside a tag or not.

**Comment by mr-martian** [comment]
Fixed in https://github.com/apertium/apertium/commit/b5916c52ea4ad196a4c84cbb42c1ca2eaf17c632 (Apertium 3.9.5)

**Comment by rmlockwood** [comment]
Will a Windows build with the fix be possible?
________________________________

**Comment by rmlockwood** [comment]
this will be fixed with the #708 PR
